staging/src/k8s.io/client-go/transport/token_source.go

```diff
@@ -156,3 +181,9 @@ func (ts *cachingTokenSource) Token() (*oauth2.Token, error) {
 	ts.tok = tok
 	return tok, nil
 }
+
+func (ts *cachingTokenSource) ResetToken() {
+	ts.RLock()
+	defer ts.RUnlock()
```

Comment:
this is writing, so it needs to take a write lock (Lock/Unlock)

Reply:
oops, done.